lib/boring_avatar/util.rb

```diff
@@ -4,10 +4,11 @@
 
 module Util
   class << self
-    def get_number(name)
-      name.chars
-        .map(&:ord)
-        .inject(:+)
+    def get_number(string)
+      string.chars.reduce(0) do |sum, char|
+        sum = (sum << 5) - sum + char.ord
+        [sum].pack("L").unpack("l").first
```

Comment:
This part converts unsigned ruby integer to signed 32 bit integer, perhaps there's a better way to do it.